### PR TITLE
Implementation of JsonReader

### PR DESCRIPTION
**Pull request by Enrique-C**
Few changes to implements JsonReader

### CODE REVIEW

File: src/main/java/salesforce/jsonfiles/config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "Admin User": {
+    "Username": "testaccount@fundacionjala.com",
+    "Password": "e7999812H"
+  },
+  "Current User": {
+    "Username": "at11.fundacion@jala.com",
+    "Password": "at11.2020"
+  }
+}
```
Comment (Amet015) [inline src/main/java/salesforce/jsonfiles/config.json]: please add a new line here

File: src/main/java/salesforce/utils/JsonFileReader.java [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 import org.json.simple.JSONObject;
 import org.json.simple.parser.JSONParser;
+import salesforce.entities.SalesforceUser;
```
Comment (Amet015) [inline src/main/java/salesforce/utils/JsonFileReader.java:14]: It's not necessary separate imports

File: src/main/java/salesforce/utils/JsonFileReader.java [hunk 3]
```diff
@@ -43,6 +55,20 @@ public static String jsonReader(final String fileName) {
...
+     * @param userType receives a String value.
+     * @return SalesforceUser object.
+     */
+    public SalesforceUser getUser(final String userType) {
+        JSONObject jsonObject = jsonReader();
+        jsonObject = (JSONObject) jsonObject.get(userType);
+        SalesforceUser salesforceUser = new SalesforceUser();
+        salesforceUser.setUsername((String) jsonObject.get(USERNAME_KEY));
+        salesforceUser.setPassword((String) jsonObject.get(PASSWORD_KEY));
+        return salesforceUser;
```
Comment (Amet015) [inline src/main/java/salesforce/utils/JsonFileReader.java]: apply ctrl + alt + L to fix the identation of the comments

File: src/main/java/salesforce/entities/SalesforceUser.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    /**
+     * Sets a Password value.
+     * @param password receives a String value.
+     */
+    public void setPassword(final String password) {
+        this.password = password;
+    }
+}
```
Comment (Amet015) [inline src/main/java/salesforce/entities/SalesforceUser.java]: apply ctrl + alt + L to fix the identation of the comments

File: src/main/java/salesforce/api/AccountApi.java [hunk 5]
```diff
@@ -35,7 +34,8 @@
      * @return response value.
      */
     public static Response postAccount(final String fileName) {
```
Comment (Amet015) [inline src/main/java/salesforce/api/AccountApi.java]: apply ctrl + alt + L to fix the identation of the comments

File: src/main/java/salesforce/utils/JsonFileReader.java (hunk 2 above)
Comment (Enrique-C) [reply src/main/java/salesforce/utils/JsonFileReader.java:14]: Its automatic when ypu use crtl+alt+l

File: src/main/java/salesforce/ui/pages/opportunity/OpportunityPageAbstract.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.ui.pages.opportunity;
+
+import salesforce.ui.pages.BasePage;
+
+/**
+ * Manages a web page.
```
Comment (Amet015) [inline src/main/java/salesforce/ui/pages/opportunity/OpportunityPageAbstract.java]: please specify what kind of web page?

File: src/main/java/salesforce/ui/components/menu/TopClassicMenu.java [hunk 7]
```diff
@@ -12,7 +12,6 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
-import salesforce.ui.pages.aplications.AllTabsClassicPage;
 
 /**
  * Manages a web page.
```
Comment (Amet015) [inline src/main/java/salesforce/ui/components/menu/TopClassicMenu.java]: please specify what kind of web page

File: src/main/java/salesforce/ui/components/menu/TopLightningMenu.java [hunk 8]
```diff
@@ -12,7 +12,6 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
-import salesforce.ui.pages.aplications.AppsLightningOrderedMenu;
 
 /**
  * Manages a web page.
```
Comment (Amet015) [inline src/main/java/salesforce/ui/components/menu/TopLightningMenu.java]: please specify what kind of web page

File: src/main/java/salesforce/ui/pages/opportunity/OpportunityLightningPage.java [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.ui.pages.opportunity;
+
+/**
+ * Manages a web page.
```
Comment (Amet015) [inline src/main/java/salesforce/ui/pages/opportunity/OpportunityLightningPage.java]: please specify what kind of web page

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 10]
```diff
@@ -9,9 +9,11 @@
...
 package salesforcetest.steps;
 
+import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import salesforce.ui.pages.PageTransporter;
 
 /**
  * Manages com.steps definition.
```
Comment (Amet015) [inline src/test/java/salesforcetest/steps/OpportunityStep.java]: com.steps?  a com package?

**Review by Amet015 — APPROVED** [review]

**Pull request merged**